libnavui-maps/api/current.txt

```diff
@@ -911,6 +921,19 @@ package com.mapbox.navigation.ui.maps.route.line.model {
     method public com.mapbox.navigation.ui.maps.route.line.model.RouteLineColorResources.Builder severeCongestionRange(kotlin.ranges.IntRange range);
   }
 
+  public final class RouteLineData {
+    method public kotlin.jvm.functions.Function0<com.mapbox.maps.extension.style.expressions.generated.Expression> getBaseExpressionProvider();
```

Comment:
Shouldn't we create an interface with a meaningful name?
At least it would look prettier for Java users.

Reply:
Done.